lib/irb/locale.rb

```diff
@@ -114,7 +114,7 @@ def load(file, priv=nil)
       if found
         unless @@loaded.include?(found)
           @@loaded << found # cache
```

Comment:
I don't think we need to keep this "cache" for performance reason. But removing it would cause a lot of redefinition warning from loading `error.rb`. So I decided to keep it for now.